app/templates/main/errorTable.html

```diff
@@ -0,0 +1,20 @@
+{% macro errorTable(tab_errors) %}
+    <table class="govuk-table" data-module="govuk-table">
```

Comment:
Can we use the Jinja macro for this instead?

Reply:
Looks good, I've switched to this way of doing it now